src/components/table-content-wrapper/table-content-wrapper.jsx

```diff
@@ -47,7 +49,9 @@ class TableContentWrapper extends Component {
       return (
         <div className={classes.root}>
           <Table tableLayoutFixed>
-            <caption className={classes.screenReadersOnly}>{localization.caption}</caption>
+            <caption className={cn(classes.caption, 'smart-table__caption', { [classes.screenReadersOnly]: captionHidden })}>
```

Comment:
"smart-table__caption" class should probably not always be applied here right?

Reply:
Correct, should not always be there